Formula/rosa-cli.rb

```diff
@@ -0,0 +1,31 @@
+class RosaCli < Formula
+  desc "RedHat OpenShift Service on AWS (ROSA) command-line interface"
+  homepage "https://www.openshift.com/products/amazon-openshift"
+  url "https://github.com/openshift/rosa.git",
+    tag:      "v1.0.1",
+    revision: "585ab60484929ffdad1f4b83ea29acca6c6e88f7",
+    shallow:  false
+  license "Apache-2.0"
+
+  head "https://github.com/openshift/rosa.git", shallow: false
+
+  livecheck do
+    url :stable
+    regex(/^v(\d+\.\d+\.\d+)$/i)
+  end
+
+  depends_on "coreutils" => :build
+  depends_on "go" => :build
+  depends_on "awscli"
+
+  def install
+    system "make"
+    bin.install "rosa"
```

Comment:
Is there no `make install` for this?

Reply:
Yes, but it just install the executable under `GOPATH`. Found easier to just `make` on the temp build folder and move to `bin`